docusaurus/docs/e2e-test-a-plugin/getting-started.md

```diff
@@ -43,6 +43,8 @@ Please refer to the [Playwright documentation](https://playwright.dev/docs/intro
 
 ## Set up `plugin-e2e`
 
+> If you would like to follow along with our example tests, use the [create-plugin](https://www.npmjs.com/package/@grafana/create-plugin) tool to generate a backend Data Source plugin.
```

Comment:
We can pass args now so could probably give this as a single command. 

i.e. `npx @grafana/create-plugin@latest  --pluginName='Sample datasource' --orgName='sample-org' --pluginDescription='This is a sample datasource.' --pluginType='datasource' --hasBackend`